dice_maiden_lite.rb

```diff
@@ -10,13 +10,16 @@
 require 'dicebag'
 require 'dotenv'
 
+@request_option = false
+
 Dotenv.load
 # Add API token
 @bot = Discordrb::Bot.new token: ENV['TOKEN'], ignore_bots: true, fancy_log: true
 
 # Check for command
 @bot.message(start_with: /^(!roll)/i) do |event|
   begin
+    check_server_options(event)
```

Comment:
I think we can leave this out of dice_maiden lite. It currently does not support sqlite interactions as its not really needed for a single instance. Some additional work would need to be done to fully support it.

Reply:
removed